frontend_vue/src/router/index.ts

```diff
@@ -24,6 +24,13 @@ const router = createRouter({
         title: 'Manage Token',
       },
     },
+    {
+      path: '/manage',
+      name: 'manage-old',
+      redirect: to => {
+        return { path: `/manage/${to.query.auth}/${to.query.token}`, query: {}}
+      },
```

Comment:
we need to add the meta title as in the other routes, as that value is used to display the page name on the header

<img width="1035" alt="Screenshot 2024-05-30 at 14 48 19" src="https://github.com/thinkst/canarytokens/assets/126554007/e906070f-f026-4768-9bef-6c3162fe93bc">
<img width="1064" alt="Screenshot 2024-05-30 at 14 48 24" src="https://github.com/thinkst/canarytokens/assets/126554007/5b6ff686-3841-49f6-b8d5-eb9e44b5de50">